recipes/opencl-icd-loader/all/conanfile.py

```diff
@@ -48,6 +48,9 @@ def layout(self):
     def requirements(self):
         self.requires(f"opencl-headers/{self.version}", transitive_headers=True)
 
+    def build_requirements(self):
+        self.tool_requires("cmake/[>=3.1 <4]")
+
```

Comment:
CCI recipes don't add cmake to build requirement themselves if cmake_minimum_required() is lower than 3.15 in upstream CMakeLists.

Reply:
could you please explain why?
I've tried to build this package (where only Conan is installed via pip) in empty docker image and it fails, because cmake is not found on the system.

Reply:
It's explained here: https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-recipes-that-use-build-tools-like-cmake-that-have-packages-in-conan-center-do-not-use-it-as-a-build-require-by-default